frontends/concrete-python/tests/execution/test_examples.py

```diff
@@ -271,3 +271,37 @@ def test_levenshtein_distance_randomly(alphabet_name, max_length, helpers):
         expected_distance = levenshtein_clear(str1, str2)
         actual_distance = levenshtein_distance.calculate(str1, str2, "simulate", show_distance=True)
         assert actual_distance == expected_distance
+
+def test_pir_basics():
```

Comment:
@umut-sahin I'm trying to test without being able to run that locally! If you can help me having something working locally, it's awesome. Or a way to test simply & quickly on the CI.